src/layouts/DefaultLayout.tsx

```diff
@@ -33,7 +34,7 @@ export default function DefaultLayout({ children }: { children: ReactNode }) {
   }, [loadBlendData, connected, walletAddress]);
 
   const faucet_pool = rewardZone.length > 0 ? rewardZone[0] : undefined;
-
+  /** @dev @TODO this may possibly be removed since it seems to be calculated everywhere */
```

Comment:
this chunk sets the `lastPoolId`.

That is, if you route to a page via the nav menu, we will assume the poolId is the last pool you routed too, or the first pool in the reward zone.

Reply:
yes the thing is this lastPool state is not read anywhere

Reply:
It's read in `NavMenu`